{{cookiecutter.service_name}}/service.py

```diff
@@ -183,7 +183,6 @@ def pre_execution_hook(self):
             self.conf["additional_parameters"]["collection_id"] = lenv.get("usid", "")
             self.conf["additional_parameters"]["process"] = "processing-results"
             self.conf["additional_parameters"]["STAGEOUT_WORKSPACE"] = self.workspace_name
-            self.conf["additional_parameters"]["STAGEIN_WORKSPACE"] = self.workspace_name
```

Comment:
Will need another argocd change to remove the input added by EODHP-315 (or change the PR)

Reply:
This input has now been removed under EODHP-425 (see [PR](https://github.com/UKEODHP/eodhp-argocd-deployment/pull/129))